src/services/TableService.js

```diff
@@ -16,9 +16,10 @@ function TableService($q, $http, URLService, UtilsService, CacheService, _, Elem
      *
      * @param {string} mmsEid SysML ID of a table instance
      */
-    var readTableCols = function(mmsEid, ws, version) {
+    var readTableCols = function(reqOb, ws, version) {
```

Comment:
is this function still needed? the code inside is still using crushinator (2.x) json schema?